pubspec.yaml

```diff
@@ -19,8 +19,8 @@ dependencies:
   mustache4dart: ^1.0.9
   package_config: '>=0.1.5 <2.0.0'
   path: ^1.3.0
-  quiver: '>=0.18.0 <0.23.0'
-  resource: ^1.1.0
+  quiver: '>=0.18.0 <0.25.0'
+  resource: ^2.1.2
```

Comment:
unless 1.1.0 no longer functions for us, better to leave the version constraint broad.